src/app/give-help/accommodation-form/accommodation-form.component.ts

```diff
@@ -44,7 +44,7 @@ export class AccommodationFormComponent {
 
   submitOffer(): void {
     this.loading = true;
-    this.phoneNumber ? this.preparePhoneNumber() : null;
+    this.phoneNumber ? this.preparePhoneNumber() : (this.data.phoneNumber = undefined);
```

Comment:
Please use regular `if`-`else` statements, expressions as statements reduce readability:
```suggestion
    if (this.phoneNumber) {
      this.preparePhoneNumber();
    } else {
      this.data.phoneNumber = undefined;
    }
```
The same remark in multiple places below.

Reply:
poprawione, zapamiętam na przyszlosc